draft-ietf-tls-rfc8447bis.md

```diff
@@ -231,58 +217,95 @@ IANA SHALL update the TLS ExtensionType Values registry as follows:
-| 0x00,0x3B | TLS_RSA_WITH_NULL_SHA256 | D |
-| 0x00,0x46 | TLS_DH_anon_WITH_CAMELLIA_128_CBC_SHA | D |
-| 0x00,0x6C | TLS_DH_anon_WITH_AES_128_CBC_SHA256 | D |
-| 0x00,0x6D | TLS_DH_anon_WITH_AES_256_CBC_SHA256 | D |
-| 0x00,0x89 | TLS_DH_anon_WITH_CAMELLIA_256_CBC_SHA | D |
 | 0x00,0x8A | TLS_PSK_WITH_RC4_128_SHA | D |
-| 0x00,0x8E | TLS_DHE_PSK_WITH_RC4_128_SHA | D |
-| 0x00,0x92 | TLS_RSA_PSK_WITH_RC4_128_SHA | D |
-| 0x00,0x9B | TLS_DH_anon_WITH_SEED_CBC_SHA | D |
-| 0x00,0xA6 | TLS_DH_anon_WITH_AES_128_GCM_SHA256 | D |
-| 0x00,0xA7 | TLS_DH_anon_WITH_AES_256_GCM_SHA384 | D |
 | 0x00,0xB0 | TLS_PSK_WITH_NULL_SHA256 | D |
 | 0x00,0xB1 | TLS_PSK_WITH_NULL_SHA384 | D |
-| 0x00,0xB4 | TLS_DHE_PSK_WITH_NULL_SHA256 | D |
-| 0x00,0xB5 | TLS_DHE_PSK_WITH_NULL_SHA384 | D |
-| 0x00,0xB8 | TLS_RSA_PSK_WITH_NULL_SHA256 | D |
-| 0x00,0xB9 | TLS_RSA_PSK_WITH_NULL_SHA384 | D |
-| 0x00,0xBF | TLS_DH_anon_WITH_CAMELLIA_128_CBC_SHA256 | D |
-| 0x00,0xC5 | TLS_DH_anon_WITH_CAMELLIA_256_CBC_SHA256 | D |
-| 0xC0,0x01 | TLS_ECDH_ECDSA_WITH_NULL_SHA | D |
-| 0xC0,0x02 | TLS_ECDH_ECDSA_WITH_RC4_128_SHA | D |
 | 0xC0,0x06 | TLS_ECDHE_ECDSA_WITH_NULL_SHA | D |
 | 0xC0,0x07 | TLS_ECDHE_ECDSA_WITH_RC4_128_SHA | D |
-| 0xC0,0x0B | TLS_ECDH_RSA_WITH_NULL_SHA | D |
-| 0xC0,0x0C | TLS_ECDH_RSA_WITH_RC4_128_SHA | D |
 | 0xC0,0x10 | TLS_ECDHE_RSA_WITH_NULL_SHA | D |
 | 0xC0,0x11 | TLS_ECDHE_RSA_WITH_RC4_128_SHA | D |
-| 0xC0,0x15 | TLS_ECDH_anon_WITH_NULL_SHA | D |
-| 0xC0,0x16 | TLS_ECDH_anon_WITH_RC4_128_SHA | D |
-| 0xC0,0x17 | TLS_ECDH_anon_WITH_3DES_EDE_CBC_SHA | D |
-| 0xC0,0x18 | TLS_ECDH_anon_WITH_AES_128_CBC_SHA | D |
-| 0xC0,0x19 | TLS_ECDH_anon_WITH_AES_256_CBC_SHA | D |
 | 0xC0,0x33 | TLS_ECDHE_PSK_WITH_RC4_128_SHA | D |
 | 0xC0,0x39 | TLS_ECDHE_PSK_WITH_NULL_SHA | D |
 | 0xC0,0x3A | TLS_ECDHE_PSK_WITH_NULL_SHA256 | D |
 | 0xC0,0x3B | TLS_ECDHE_PSK_WITH_NULL_SHA384 | D |
-| 0xC0,0x46 | TLS_DH_anon_WITH_ARIA_128_CBC_SHA256 | D |
-| 0xC0,0x47 | TLS_DH_anon_WITH_ARIA_256_CBC_SHA384 | D |
-| 0xC0,0x5A | TLS_DH_anon_WITH_ARIA_128_GCM_SHA256 | D |
-| 0xC0,0x5B | TLS_DH_anon_WITH_ARIA_256_GCM_SHA384 | D |
-| 0xC0,0x84 | TLS_DH_anon_WITH_CAMELLIA_128_GCM_SHA256 | D |
-| 0xC0,0x85 | TLS_DH_anon_WITH_CAMELLIA_256_GCM_SHA384 | D |
 | 0xC0,0xB4 | TLS_SHA256_SHA256 | D |
 | 0xC0,0xB5 | TLS_SHA384_SHA384 | D |
 
-
 - Update note on the Recommended column with text in {{rec-note}}.
 
+- {{!I-D.ietf-tls-deprecate-obsolete-kex}} sets the Recommended column
+  for the following cipher suites to "D". This document adds additional
+  references to indicate why the cipher suites are marked as "D".
+
+<aside markdown="block">
+  RFC Editor NOTE: Replace [ThisRFC] in the table that follows with the
+  eventual RFC #.
+</aside>
+
+| Value | Cipher Suite Name                            | Reference |
+|:------|:---------------------------------------------|----------:|
+| 0x00,0x01 | TLS_RSA_WITH_NULL_MD5                    | [ThisRFC] |
```

Comment:
Should this column reference both obsolete keyex and this doc?  [ThisRFC],[ObsoleteKeyxRFC]

Maybe just change the Replace[ThisRFC] with the RFC numbers for this document and the ietf-tls-deprecate-obsolete-kex document

Reply:
Sounds good to me.